test/unit/cors_test.rb

```diff
@@ -55,7 +55,7 @@ def load_app(name)
       eval File.read(File.dirname(__FILE__) + "/#{name}.ru")
       map('/') do
         run proc { |env|
-          [200, {'Content-Type' => 'text/html'}, ['success']]
+          [200, {'Content-Type' => 'text/html'}.freeze, ['success'].freeze]
```

Comment:
`['success'].freeze` is not related to the fix in this PR, but just in case...

Reply:
@uasi Thank you for this!  Can you do me a favor and create a specific test for this?  I'd want to make sure we document (in a test) that we support this explicitly.

Thanks!
Calvin